crates/nostr/src/nips/nip11.rs

```diff
@@ -6,6 +6,8 @@
 //!
 //! <https://github.com/nostr-protocol/nips/blob/master/11.md>
 
+use crate::types::time::Timestamp;
```

Comment:
Sorry for nitpicking, can you move this under the external dependencies import?

In my code I use always this import struct:

* `core`, `alloc` and `std` at the top
* external deps in the middle
* internal modules at the bottom

So should be:

```rust
use alloc::string::String;
use alloc::vec::Vec;
use core::fmt;
#[cfg(not(target_arch = "wasm32"))]
use std::net::SocketAddr;

#[cfg(not(target_arch = "wasm32"))]
use reqwest::Proxy;
use url_fork::Url;

use crate::types::time::Timestamp;
```

The rust fmt feature to automate this is unstable so for now I can't enable it